lib/ClangImporter/ClangImporter.cpp

```diff
@@ -934,7 +934,9 @@ importer::addCommonInvocationArguments(
         invocationArgStrs.push_back("-isystem");
         invocationArgStrs.push_back(path.Path);
       } else {
-        invocationArgStrs.push_back("-I" + path.Path);
+        llvm::SmallString<261> Path{path.Path};
+        llvm::sys::path::make_preferred(Path);
```

Comment:
Don't we need to do this for the path strings above too?

Reply:
Yes, I missed that in this revision, but I didn't want to interrupt the build itself.